GAPS (#47)
msupply-foundation/msupply_docs: content/docs/manage/demographics.md

+++
title = "Demographics"
description = "Demographic indicators"
date = 2022-05-17T16:20:00+00:00
updated = 2022-05-17T16:20:00+00:00
draft = false
weight = 3
sort_by = "weight"
template = "docs/page.html"

[extra]
toc = true
top = false
+++

> Demographics is available only on the [Open mSupply Central Server](/docs/getting_started/central). This is where you can view and manage future projections for different demographic indicators.

Reviewer: is? are? darn it - is this a plural noun or a menu section which is singular??
Perhaps `The demographics section is only...` 🤷

> Demographics is available only on the [Open mSupply Central Server](/docs/getting_started/central). This is where you can view and manage future projections for different demographic indicators.

Reviewer: ```suggestion
You can manage demographics only on the [Open mSupply Central Server](/docs/getting_started/central). This is where you can view and manage future projections for different demographic indicators. This population data can then be used to estimate the cold storage capacity which will be required for upcoming immunization programs.
```

## Configuration

> Currently, `Demographics` features are only enabled as part of the `Vaccine` module.

Reviewer: ```suggestion
Currently, `Demographics` features are enabled as part of the `Vaccine` module.
```

no need for 'only' and it implies something is lacking

> To begin, enable the store preference for `mobile: Uses Vaccine Module` (see the [store preferences](https://docs.msupply.org.nz/other_stuff:virtual_stores#preferences_tab) documentation for how to do this).

Reviewer: nice! that's handy to have in here

## Viewing Demographics Indicators

Choose `Manage` > `Demographics` in the navigation panel.

![goto: demographics](/docs/manage/images/goto_demographics.png)

> You will be presented with a list of Demographics Indicators:

Reviewer: :nit the use of 'list' would mean that the item listed is likely singular
```suggestion
You will be presented with a list of Demographic Indicators:
```

![indicators](/docs/manage/images/indicators.png)

> The list of Demographics Indicators is divided into 8 columns:

Reviewer: :nit
```suggestion
The list of Demographic Indicators is divided into 8 columns:
```
